cmd/security-agent/subcommands/runtime/security_profile.go

```diff
@@ -78,3 +91,158 @@ func showSecurityProfile(log log.Component, config config.Component, activityDum
 
 	return nil
 }
+
+func listSecurityProfileCommands(globalParams *command.GlobalParams) []*cobra.Command {
+	cliParams := &securityProfileCliParams{
+		GlobalParams: globalParams,
+	}
+
+	securityProfileListCmd := &cobra.Command{
+		Use:   "list",
+		Short: "get the list of active security profiles",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return fxutil.OneShot(listSecurityProfiles,
+				fx.Supply(cliParams),
+				fx.Supply(core.BundleParams{
+					ConfigParams: config.NewSecurityAgentParams(globalParams.ConfigFilePaths),
+					LogParams:    log.LogForOneShot(command.LoggerName, "info", true)}),
+				core.Bundle,
+			)
+		},
+	}
+
+	securityProfileListCmd.Flags().BoolVar(
+		&cliParams.includeCache,
+		flags.IncludeCache,
+		false,
+		"defines if the profiles in the Security Profile manager LRU cache should be returned",
+	)
+
+	return []*cobra.Command{securityProfileListCmd}
+}
+
+func listSecurityProfiles(log log.Component, config config.Component, args *securityProfileCliParams) error {
+	client, err := secagent.NewRuntimeSecurityClient()
+	if err != nil {
+		return fmt.Errorf("unable to create a runtime security client instance: %w", err)
+	}
+	defer client.Close()
+
+	output, err := client.ListSecurityProfiles(args.includeCache)
+	if err != nil {
+		return fmt.Errorf("unable send request to system-probe: %w", err)
+	}
+	if len(output.Error) > 0 {
+		return fmt.Errorf("security profile list request failed: %s", output.Error)
+	}
+
+	if len(output.Profiles) > 0 {
+		fmt.Println("security profiles:")
+		for _, d := range output.Profiles {
+			printSecurityProfileMessage("  ", d)
```

Comment:
as this func is only called here with the prefix constant, maybe we can move it inside the func itself?

Reply:
done 👍🏻